src/ServicePulse.Host/vue/src/composables/isRouteSelected.ts

```diff
@@ -0,0 +1,15 @@
+import { useLink, useRoute } from "vue-router";
+
+export default function isRouteSelected(path: string) {
+  const route = useRoute();
+
+  if (route.path === path) {
+    return true;
+  }
+
+  if (`${path}/` === route.path) {
```

Comment:
This is to handle the scenarios when the child route path is an empty string ("").
Eg, `/configuration` vs `/configuration/`
See https://github.com/Particular/ServicePulse/blob/b27138fc4e28f82cacbf414dbc9d908c3b75663a/src/ServicePulse.Host/vue/src/router/routeLinks.ts#L26